script/collect.ts

```diff
@@ -1,33 +1,43 @@
-#!/usr/bin/env node
-
-require('dotenv-safe').load()
-
-const del = require('del')
+#!/usr/bin/env ts-node
+
+import * as del from 'del'
+import * as fs from 'fs'
+import * as got from 'got'
+import { sync as mkdir } from 'make-dir'
+import * as path from 'path'
+import { execSync } from 'child_process'
+import * as Octokit from '@octokit/rest';
 const electronDocs = require('electron-docs')
-const fs = require('fs')
-const got = require('got')
-const mkdir = require('make-dir').sync
-const path = require('path')
-const { execSync } = require('child_process')
-const github = require('@octokit/rest')()
 const englishBasepath = path.join(__dirname, '..', 'content', 'en-US')
 
-if (process.env.GH_TOKEN) {
-  github.authenticate({
-    type: 'token',
-    token: process.env.GH_TOKEN
-  })
+const github = new Octokit({
+  auth: process.env.GH_TOKEN ? process.env.GH_TOKEN : ''
+})
+
+interface IResponse {
+  tag_name: string
+  assets: Octokit.ReposGetReleaseByTagResponseAssetsItem[]
 }
 
-let release
+interface IElectronDocsResponse {
+  slug: string
+  filename: string
+  markdown_content: string
+}
+
+let release: IResponse
 
-del(englishBasepath)
-  .then(fetchRelease)
-  .then(fetchAPIDocsFromLatestStableRelease)
-  .then(fetchApiData)
-  .then(getMasterBranchCommit)
-  .then(fetchTutorialsFromMasterBranch)
-  .then(fetchWebsiteContent)
+main()
```

Comment:
Will this crash with a non-zero exit code if it rejects? If not, let's add a `.catch(err => ...)` here